### PR TITLE
target label added on NSX-T collectors

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,8 +3,9 @@ prometheus-client
...
 PyYAML
-Flask==1.1.1
+Flask==1.1.2
 gevent
 cffi
 beautifulsoup4==4.9.3
-lxml
\ No newline at end of file
+lxml
+itsdangerous==1.1.0
```
Comment (viennaa) [inline requirements.txt:11]: This is not being used, is it?

Comment (richardtief) [reply requirements.txt:11]: Look at the failing tests above. This version looks like a needed dependency.

Comment (richardtief) [reply requirements.txt:11]: At least for the PyTest.

Comment (viennaa) [reply requirements.txt:11]: Ok, implicit dependency through Flask, I see.

**Review by viennaa — APPROVED** [review]
Let's try this again.

**Pull request merged**